### PR TITLE
Pipeline for deploying documentation

### PR DESCRIPTION
**Pull request by geramirez**
Auto deploy pipeline for compliance.cloud.gov

### CODE REVIEW

File: deploy-compliance-masonry/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - put: deploy-documentation
+    params:
+      manifest: create-gitbook/run-compliance-masonry/compliance-repository/manifest.yaml
+      path: create-gitbook/run-compliance-masonry/compliance-repository
+
+resources:
+- name: compliance-repository
+  type: git
+  source:
+    uri: https://github.com/18F/cg-compliance.git
```
Comment (LinuxBozo) [inline deploy-compliance-masonry/pipeline.yml:36]: Pretty sure you have to also specify a `branch:`

Comment (geramirez) [inline deploy-compliance-masonry/pipeline.yml:36]: I'm assuming master works

**Comment by LinuxBozo** [comment]
@geramirez Might want to update the `credentials.yml.example` with example values?

**Comment by geramirez** [comment]
@LinuxBozo done.

**Comment by LinuxBozo** [comment]
:+1:

**Pull request merged**